version.json

```diff
@@ -0,0 +1,3 @@
+{
+    "version": "v0.0.1"
```

Comment:
Ready to tag already? I recommend excluding this, set up unified CI properly and have a version deployed to test it's all good before tagging a version.